modules/installation/pages/2-pre-installation/1-usb-creation.adoc

```diff
@@ -0,0 +1,186 @@
+= USB Creation
+:toc:
+:toclevels: 2
+
+This guide walks the user through setting up a bootable USB for bootstrapping the Fawkes cluster.
```

Comment:
This guide walks the user through setting up a bootable USB drive for bootstrapping the Fawkes cluster.